README.md

```diff
@@ -36,6 +36,15 @@ Convert Juptyer notebook file, `mynotebook.ipynb`, to HTML using:
 
 This command creates an HTML output file named `mynotebook.html`.
 
+## Dev Install
+Install nbconvert for development using:
+
+```
+git clone https://github.com/jupyter/nbconvert.git
+cd nbconvert
+python setup.py develop
```

Comment:
`pip install -e .`  is preferred IIRC, (maybe even `python -m pip install -e .` but that's another story). Because if you don't use pip, pip might not be able to correctly uninstall later. In general you should never invoke setup.py directly.